decoration.sk

```diff
@@ -331,3 +323,96 @@ update aquatic:
 		(trio of|[bundle of] three) = -[pickles=3]
 		[bundle of] four = -[pickles=4]
 	({waterlogged} {pickle count}|{pickle count} {waterlogged}) sea pickle¦s = minecraft:sea_pickle
+	
+unchanged orientations:
+	{skull floor orientations}:
+		north[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:0}
+		north-northeast[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:1}
+		northeast[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:2}
+		east-northeast[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:3}
+		east[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:4}
+		east-southeast[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:5}
+		southeast[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:6}
+		south-southeast[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:7}
+		south[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:8}
+		south-southwest[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:9}
+		southwest[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:10}
+		west-southwest[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:11}
+		west[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:12}
+		west-northwest[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:13}
+		northwest[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:14}
+		north-northwest[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:15}
+	{banner standing orientations}:
+		south[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:0}
+		south-southwest[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:1}
+		southwest[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:2}
+		west-southwest[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:3}
+		west[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:4}
+		west-northwest[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:5}
+		northwest[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:6}
+		north-northwest[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:7}
+		north[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:8}
+		north-northeast[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:9}
+		northeast[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:10}
+		east-northeast[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:11}
+		east[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:12}
+		east-southeast[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:13}
+		southeast[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:14}
+		south-southeast[ward[s]][( |-)rotat(ion|ed)] = - {Damage:1,Rot:15}
+		
+old orientations:
+	minecraft version = 1.12.2 or older
+	{skull types}:
+		skelet(on|al) (skull|head)¦s = - {SkullType:0} 
+		wither skle(on|al) (skull|head)¦s = - {SkullType:1}
+		zombi(e|fied) (head|skull)¦s = - {SkullType:2}
+		player (head|skull)¦s = - {SkullType:3}
+		creeper (head|skull)¦s = - {SkullType:4}
+	{wall orientations}:
+		north[ward[s]][( |-)(facing|pointing)] = - {Damage:3}
+		south[ward[s]][( |-)(facing|pointing)] = - {Damage:2}
+		east[ward[s]][( |-)(facing|pointing)] = - {Damage:4}
+		west[ward[s]][( |-)(facing|pointing)] = - {Damage:5}
+	
+combat update skulls:
+	minecraft version = 1.9 or newer
```

Comment:
Oh, last thing, this should be 1.9 to 1.12.2 since you redefine ender dragon heads for 1.13 down below.

Reply:
Oh, right. Fixed.